tools/sentencepiece_testing/create_phi3_test_proto.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+
+import sentencepiece.sentencepiece_model_pb2 as sp_pb2
+
+from tools.sentencepiece_testing.utils import train_sentencepiece
+
+ADDED_TOKENS = [
+    "<|endoftext|>",
+    "<|assistant|>",
+    "<|placeholder1|>",
```

Comment:
No harm in these, but in general, you can usually make a tiny sentencepiece model with only the tokens that we actually need to set up our tokenizer, etc.

AFAIK, nothing will actually need these placeholders.

Reply:
Yes I agree but I just copied all the special tokens from huggingface model repo.